Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -202,6 +202,13 @@ void Start()
             SetMelee(ScreenWeapon);
         }
 
+        bool IsHoldAttackMode()
+        {
+            return DaggerfallUnity.Settings.AttackMode == 2 && InputManager.Instance.GetKey(
```

Comment:
Nitpick, the function could be "IsHoldingAttackMode" I think. The current name made me think it was checking for attack mode, but it's also checking for input being held

Reply:
You're right. What method actually does is `IsHoldingHoldAttackMode`. I've moved the button held check to InputManager.